### PR TITLE
Update repo to v1.12.32

### PR DESCRIPTION
**Pull request by Realignist**
I update repo to v1.12.32.

### CODE REVIEW

File: Library/Formula/repo.rb [hunk 1]
```diff
@@ -1,9 +1,9 @@
 class Repo < Formula
   desc "Repository tool for Android development"
   homepage "https://source.android.com/source/developing.html"
-  url "https://raw.githubusercontent.com/android/tools_repo/v1.12.21/repo"
-  version "1.21"
-  sha256 "44d1ac8b948e6154a495469dad05870dc6182712c0610da8d255b5471dd5bcb9"
+  url "https://raw.githubusercontent.com/android/tools_repo/v1.12.32/repo"
+  version "1.12.32"
+  sha256 "9907c36d63cf5222d73ab270fee5249dc84f6b4580cda03ddd7ac309f11b289e"
   revision 1
```
Comment (bfontaine) [inline Library/Formula/repo.rb]: Please remove this `revision` line.

**Comment by bfontaine** [comment]
We might have an issue with the version here; Homebrew will assume `1.12.32` < `1.21`.

**Comment by Realignist** [comment]
@bfontaine I just removed the version of this file.

File: Library/Formula/repo.rb [hunk 2]
```diff
@@ -1,9 +1,8 @@
 class Repo < Formula
   desc "Repository tool for Android development"
   homepage "https://source.android.com/source/developing.html"
-  url "https://raw.githubusercontent.com/android/tools_repo/v1.12.21/repo"
-  version "1.21"
-  sha256 "44d1ac8b948e6154a495469dad05870dc6182712c0610da8d255b5471dd5bcb9"
+  url "https://raw.githubusercontent.com/android/tools_repo/v1.12.32/repo"
+  sha256 "9907c36d63cf5222d73ab270fee5249dc84f6b4580cda03ddd7ac309f11b289e"
   revision 1
```
Comment (bfontaine) [inline Library/Formula/repo.rb]: Please remove this `revision` line as well.

**Comment by bfontaine** [comment]
@Realignist Please leave it; Homebrew can’t parse it from the URL here. The problem we have here is that the previous version was “`1.21`” and the new one is “`1.12.32`”; Homebrew will assume that the new version is not higher than the previous one and won’t upgrade it.
We might want to keep with the [original version numbering](https://github.com/Homebrew/homebrew/pull/23107/files) and use `version "1.22"` to avoid the issue.

**Comment by Realignist** [comment]
Okay, I understand.

File: Library/Formula/repo.rb [hunk 3]
```diff
@@ -1,9 +1,9 @@
 class Repo < Formula
   desc "Repository tool for Android development"
   homepage "https://source.android.com/source/developing.html"
-  url "https://raw.githubusercontent.com/android/tools_repo/v1.12.21/repo"
-  version "1.21"
-  sha256 "44d1ac8b948e6154a495469dad05870dc6182712c0610da8d255b5471dd5bcb9"
+  url "https://raw.githubusercontent.com/android/tools_repo/v1.12.32/repo"
+  version "1.22"
+  sha256 "9907c36d63cf5222d73ab270fee5249dc84f6b4580cda03ddd7ac309f11b289e"
   revision 1
```
Comment (bfontaine) [inline Library/Formula/repo.rb]: Please remove the line with `revision 1`. Revisions are reset when we upgrade the version; so this line is not necessary anymore.

**Comment by Realignist** [comment]
Sorry for late.

**Comment by bfontaine** [comment]
Thank you for your contribution @Realignist; we appreciate it! :tada:

**Pull request closed without merging**